package.json

```diff
@@ -5,6 +5,9 @@
   "description": "Code du travail",
   "homepage": "https://github.com/SocialGouv/code-du-travail-numerique#readme",
   "scripts": {
+    "postinstall": "husky install",
+    "prepublishOnly": "pinst --disable",
```

Comment:
since the package is private, we probably don't need `pinst` and pre/post publish

Reply:
ah maiis oui j'ai confondu private et open source je crois ^^'

Reply:
We have some sub repos that are not private but I guess that's ok